exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -126,8 +139,11 @@ func (e *exporter) Capabilities() consumer.Capabilities {
 }
 
 func (e *exporter) Shutdown(ctx context.Context) error {
-	if e.pusher != nil {
-		e.pusher.ForceFlush()
+	for _, pusher := range e.listPushers() {
+		err := pusher.ForceFlush()
+		if err != nil {
+			return err
+		}
 	}
 	return nil
 }
```

Comment:
All of the pushers should be forced to flush before returning from the function.

```suggestion
func (e *exporter) Shutdown(ctx context.Context) (errs error) {
    for _, pusher := range e.ListPushers() {
        errs = mutlierr.Append(errs,  pusher.ForceFlush())
    }
    return errs
}
```